css/style.css

```diff
@@ -92,3 +92,313 @@ nav {
 ul li {
   list-style-type: none;
 }
+
+.action-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 64px;
+  margin-bottom: 22px;
+}
+
+.recent-work header {
+  display: grid;
+  justify-content: center;
+}
+
+.recent-work h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  margin-left: 26px;
+  margin-right: 22px;
+  margin-top: 12px;
+}
+
+.indicator-line {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+  margin-bottom: 62px;
+}
+
+.card-area {
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  margin-left: 26px;
+  margin-right: 22px;
+  margin-top: 62px;
+}
+
+.card-details {
+  margin-top: 252px;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.card-details h3 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  color: #3a4a42;
+  padding-top: 16px;
+}
+
+.card-tag {
+  display: flex;
+  flex-direction: row;
+  margin-top: 16px;
+  margin-bottom: 29px;
+}
+
+.tags {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.card-tag .ruby {
+  margin-left: 6px;
+  margin-right: 4px;
+}
+
+.card-tag .css {
+  margin-left: 4px;
+  margin-right: 4px;
+}
+
+.card-tag .javascript {
+  margin-left: 4px;
+  margin-right: 4px;
+}
+
+.card-tag .html {
+  margin-left: 4px;
+  margin-right: 6px;
+}
+
+.btn {
+  display: flex;
+  justify-content: center;
+}
+
+.button {
+  background: #36b37f;
+  border-radius: 4px;
+  width: 122px;
+  height: 48px;
+  color: #fff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+  margin-bottom: 29px;
+}
+
+.button:hover {
+  background: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  border-radius: 4px;
+}
+
+.button:enabled {
+  background: #36b37f;
+  border-radius: 4px;
+}
+
+.button:active {
+  background: #008552;
+  border-radius: 4px;
+}
+
+.about-title {
+  text-align: center;
+}
+
+.about-me {
+  margin-top: 58px;
+}
+
+.about-me h3 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.about-text {
+  display: block;
+  position: relative;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+  text-align: center;
+  margin: 24px 12px;
+}
+
+.btns {
+  width: 159px;
+  height: 48px;
+}
+
+.about-img {
+  position: relative;
+  float: right;
+  margin: -20px;
+  top: -70px;
+}
+
+.about-imgs {
+  position: relative;
+  float: left;
+  margin: -10px;
+  top: -70px;
+}
+
+.lang {
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  width: 327px;
+  height: 352px;
+  margin-top: 59px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
```

Comment:
- [ ] I would recommend you consider wrapping the technologies cards in a container and style using display: flex to layout the content of each card and make the section responsive 📱

Reply:
I worked on this